api.js

```diff
@@ -4,11 +4,14 @@ const api = axios.create({
   baseURL: "https://kyle-wearing-be-nc-news.onrender.com/api",
 });
 
-export function getArticles(page) {
+export function getArticles(page, topic) {
   let queryStr = "?";
   if (page) {
     queryStr += `p=${page}`;
   }
+  if (topic) {
+    queryStr += `&topic=${topic}`;
+  }
```

Comment:
Axios has a nicer way to handle optional params. Have a look at the optional `params` object as that will make this much nicer. It will also help you on the next ticket!